build_dnssec_prover.py

```
@@ -4,12 +4,15 @@
 import shutil
 from pathlib import Path
 
+
 def build():
     target_dir = Path(os.path.dirname(__file__)).joinpath("src/utils/dnssec")
 
     build_dir = Path(os.path.dirname(__file__)).joinpath("dnssec-prover/wasmpack")
     os.chdir(build_dir)
-    os.system("wasm-pack build --target web --release && rm Cargo.lock && rm -r target/")
+    os.system(
```

Ruff (0.15.10)

[error] Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)